app/components/participant_status_tag_component.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class ParticipantStatusTagComponent < BaseComponent
+  def initialize(profile:, admin: false)
+    @profile = profile
+    @admin = admin
+  end
+
+  def call
+    if profile.npq?
+      render Admin::Participants::NPQValidationStatusTag.new(profile: profile)
+    else
+      govuk_tag tag_attributes
+    end
```

Comment:
I sense a disturbing lack of symetry. Do we want a `Admin::Participants::ECFValidationStatusTag`?

Reply:
No, because this isn't just for admins

Reply:
The asymmetry here is that there is no SIT equivalent for NPQ

Reply:
I meant more that NPQ get a component to cover their tag attributes, but ECF's tag attributes are handled in the general component

Reply:
I can extract if you'd prefer. I was mostly leaving the NPQ stuff alone because I expect it will need to change soon anyway

Reply:
Hmm, alright. I would hope there will be more in common than different between them eventually.

Reply:
They should end up pretty similar, but not identical